tests/unit/branch_name_test.py

```diff
@@ -0,0 +1,60 @@
+import branch_name_verification as branch_name_verification
+from branch_name_verification import is_excluded_branch, get_issue_num, has_issue_num, has_invalid_symbols, has_double_hyphen, has_branch_name_error
+
+excluded_branches = {'test-1', 'sdfsdfs', '!@dvsfosk*&*&^fs4943'}
+
+def test_excluded_branches():
+    """
+    test excluded branches checking
+    """
+    assert is_excluded_branch('test', excluded_branches) == False
+    assert is_excluded_branch('test-1', excluded_branches) == True
+    assert is_excluded_branch('!@dvsfosk*&*&^fs4943', excluded_branches) == True
+
+def test_get_issue_num():
```

Comment:
missing docstring

Reply:
fixed

Reply:
I'm nitpicking but reading that docstring kind of confuses me but "test the checking of excluded branches" sounds better

Reply:
fixed